map/mwm_url.cpp

```diff
@@ -176,22 +172,19 @@ ParsedMapApi::UrlType ParsedMapApi::SetUrlAndParse(string const & raw)
          ParseCommonParam(key, value);
       });
 
-      guard.release();
       return m_requestType = UrlType::Crosshair;
     }
     else if (checkForGe0Link)
     {
       // The URL is prefixed by one of the kGe0Prefixes AND doesn't match any supported API call:
       // => try to decode ge0 short link.
-      ge0::Ge0Parser parser;
       ge0::Ge0Parser::Result info;
-      if (!parser.ParseAfterPrefix(raw, prefix, info))
+      if (ge0::Ge0Parser parser; !parser.ParseAfterPrefix(raw, prefix, info))
```

Comment:
I don't like this syntax here. I suppose that if (ge0::Ge0Parser().ParseAfterPrefix(...)) will be enough here, but declaring vars in if looks strange IMO.